analyses/molecular-subtyping-pathology/pathology-subtyping-craniopharyngioma.Rmd

```diff
@@ -0,0 +1,121 @@
+---
+title: "Recoding adamantinomatous craniopharyngiomas"
+output: 
+  html_notebook:
+    toc: true
+    toc_float: true
+author: JN Taroni for ALSF CCDL (code)
+date: 2021
+---
+
+_Background adapted from [#994](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/994)_
+
+There are Craniopharyngioma samples which may have been annotated as "To be classified" in `molecular-subtyping-CRANIO` because they lack canonical mutations.
+However, for adamantinomatous craniopharyngioma, the b-catenin SNV is not present in all samples ([ref](https://doi.org/10.1093/jnen/nlw116)):
+
+> In our cohort of [adamantinomatous craniopharyngiomas] specimens from 117 patients we found _CTNNB1_ mutations in 89 cases (76.1%).
+
+There are samples described as `Adamantinomatous` in pathology reports, so we can update the `harmonized_diagnosis` and `molecular_subtype` information accordingly.
+
+## Set up
+
+### Libraries
+
+```{r}
+library(tidyverse)
+```
+
+### Input
+
+```{r}
+data_dir <- file.path("..", "..", "data")
+histologies_file <- file.path(data_dir, "pbta-histologies.tsv")
```

Comment:
Should this be using `pbta-histologies.tsv` or `pbta-histologies-base.tsv`?

Reply:
I think it should be pbta-histologies-base.tsv here as well, since in the process of subtyping only `pbta-histologies-base.tsv` file is available for the scripts in molecular-subtyping-pathology.

Reply:
My understanding is all subtypes assignment/update should be done on pbta-histologies-base.tsv and in `molecular-subtying-integrate` we combined the pbat-histologies-base.tsv with the molecular subtypes from various OpenPBTA modules to create `pbta-histologies.tsv`.

Reply:
Hmm... I think we probably want to use `pbta-histologies-base.tsv` + the output of the `01` notebook in this module, because these steps are downstream of CRANIO subtyping. This may speak to @jharenza's point in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1016#pullrequestreview-642843663. I can revise to use `pbta-histologies-base.tsv` and the compiled subtypes file from `01`.

I still think making a final(ish) decision on organization is premature.

Reply:
I agree! Can we created a new output file in molecular-subtyping-pathology to capture the pathology_free_text_diagnoiss subtype updates?
In an update #1047 we after discussing with @jharenza we added a file `compiled_molecular_subtypes_with_clinical_pathology_feedback_and_report_info.tsv` which is a combined file with molecular_subtyping ( from 01 ) + clinical feedback(from 02) + pathology feedback (03) + pathology_free_text_diagnosis report information for meningioma only so that we can then use the `compiled_molecular_subtypes_with_clinical_pathology_feedback_and_report_info.tsv` as input for `molecular-subtyping-integrate`. 

Do you think we can save the CRANIO subtyping updates in the similar logic as well?

Reply:
See what you think of https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1016/commits/0fa0551d0dffab30a1e41652dd6d44c9c6b5fd3b.

I am somewhat of the opinion that #1047 is the kind of thing that should wait until all of the molecular subtyping pathology stuff goes through and we can assess what to do with the module's organization more holistically. As I understand it, these subtyping efforts are not blocking v19 because the purpose of v19 is mostly to get out the MAF changes, remove an RNA-seq sample, and a few histologies file changes.

Reply:
I had to update the code because in the master repo the script was using`pbta-histologies.tsv` which would error out because the file is not available for scripts in `molecular-subtyping-pathology` and pbta-histologies-base.tsv doesn't have `harmonised_diagnosis`.
 I had it commented out https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1028#discussion_r623185427 but then we decided on updating the code to include this update in v19.

Reply:
Hi @jaclyn-taroni! I am just catching up here. Are you ok with #1047, which as it stands would need to be merged before the rest of the release goes out for v19 based on the use of `pbta-histologies.tsv` instead of `pbta-histologies-base.tsv`? Otherwise, we could break up the PR and work on meningioma later, but I would also prefer to get v19 out asap so we can then finish up and get v20 out soon :).

Reply:
Yea, I'm good with #1047 as is. We can re-evaluate after v19 goes out and we keep working through all the molecular subtyping pathology issues.

Reply:
great, thanks!